_git/_git.py

```diff
@@ -84,13 +70,15 @@ def __init__(
         super(GitCommandRule, self).__init__(
             name=name,
             spec='[help] ({}) <options>'.format(alias),
+            exported=False,
             extras=[Repetition(
                 name='options',
                 min=0,
                 max=10,
                 child=RuleRef(MappingRule(
                     name=name + '_options',
-                    mapping=wrap_options(options),
+                    mapping=options,
+                    exported=False,
```

Comment:
before, the problem was that you could say stuff like `stash`  instead of `git stash`  and Dragon would type out `stash`  which is unexpected.

 it seems that the exported false option works well on the new version of dragonfly2 o prevent this.